mne/time_frequency/tfr.py

```diff
@@ -185,6 +185,9 @@ def _cwt_fft(X, Ws, mode="same", decim=1):
     """Compute cwt with fft based convolutions
     Return a generator over signals.
     """
+    if mode not in ['same', 'valid', 'full']:
```

Comment:
Since I changed L211, the mode isn't explicit. I added the test to re-explicit the mode.